packages/cashier/globals.d.ts

```diff
@@ -1,4 +1,2 @@
 declare module '@deriv/components';
```

Comment:
Just for my understanding why you didn't remove the whole file? is there any problem here?

Reply:
there are some errors that are coming from components workspace so we need to fix them from that side which is converting those files to typescript. that's why we decide to keep these inside declaration file for now until we migrate other packages to typescript and then comeback and remove it from here.

Reply:
@iman-fs Wouldn’t that be double work then? In my opinion it would be better to have the right config and see the errors instead of hiding them 🤔

Reply:
@farzin-fs I should check it with @ali-fs :thinking:

Reply:
for now that we are not running `tsc` when commiting new changes (like what we have for eslint, stylelint, etc) It would be no problem having TS errors.but let's hear ako's opinion on this too.

Reply:
yess,we just wanted to have a clean cli, and we started to do components and shared packages ts migration.
hopefully we can have them by the end of this month.